packages/text-field/helper-text/README.md

```diff
@@ -30,6 +30,39 @@ role | String | Sets role on HTML element.
 showToScreenReader | Boolean | toggles the aria-hidden property to allow element to be visible to the screen reader.
 validation | Boolean | If true, alters the helper text to an error message.
 
+## Input Validation
+
+HelperText provides validity styling by setting the `validation` prop in HelperText. Validation can be checked through the appropriate `Input` validation properties (pattern, min, max, require, step, minlength, maxlength).
```

Comment:
require --> required

Reply:
capitalize minlength/maxlength --> minLength/maxLength

Reply:
Should have reviewed those props, will change.